BP/history/provinces/bp_k_septem_provincae.txt

```diff
@@ -1,40 +1,68 @@
-﻿#k_septem_provincae
+﻿# Notes:
+# Aqutitanian language extinct by Early Middle Ages (so ca. 700 by the Mod's reckoning)
```

Comment:
Minor quibble, typo, Aquitanian

Reply:
Ah English; orthography and pronunciation have absolutely nothing to do with each other.